crates/api/src/service.rs

```diff
@@ -45,7 +45,18 @@ impl ApiService {
 
         let db = Arc::new(postgres_db);
 
-        let builder_infos = db.get_all_builder_infos().await.expect("failed to load builder infos");
+        let mut builder_infos = db.get_all_builder_infos().await.expect("failed to load builder infos");
+
+        builder_infos.push(BuilderInfoDocument {
+            pub_key: PublicKey::try_from(
+                Bytes::from_hex("0xaa1488eae4b06a1fff840a2b6db167afc520758dc2c8af0dfb57037954df3431b747e2f900fe8805f05d635e9a29717b").unwrap().as_ref()
+            ).expect("failed to convert to public key"),
+            builder_info: BuilderInfo {
+                collateral: U256::MAX / U256::from(4),
+                is_optimistic: true,
+                builder_id: Some("Bolt".to_string())
+            }
+        });
```

Comment:
This is currently hardcoded to avoid inserting builders manually each time we start the devnet via an SQL query. A configuration-based approach is explored in this PR: https://github.com/gattaca-com/helix/pull/41. For now we can keep it as it this and change it later.